pro/src/new_components/RouteLeavingGuardOfferCreation/RouteLeavingGuardOfferCreation.tsx

```diff
@@ -25,27 +27,28 @@ const RouteLeavingGuardOfferCreation = ({
       const stocksPathRegex = isCollectiveFlow
         ? /\/offre\/((T-){0,1}[A-Z0-9]+)\/collectif\/stocks/g
         : /\/offre\/([A-Z0-9]+)\/individuel\/stocks/g
+
+      const visibilityPathRegex =
+        /\/offre\/((T-){0,1}[A-Z0-9]+)\/collectif\/visibilite/g
+
       const confirmationPathRegex = isCollectiveFlow
         ? /\/offre\/((T-){0,1}[A-Z0-9]+)\/collectif\/confirmation/g
         : /\/offre\/([A-Z0-9]+)\/individuel\/confirmation/g
 
-      if (
-        (location.pathname.match(stocksPathRegex) &&
-          nextLocation.pathname.startsWith(offerCreationPath)) ||
-        (location.pathname.match(offerCreationPath) &&
-          nextLocation.pathname.match(confirmationPathRegex))
-      ) {
-        redirectPath = '/offres'
-        return { redirectPath, shouldBlock: true }
-      }
+      // going from confirmation to stock
```

Comment:
j'ai l'impression qu'il manque le "from stock to offer" ? (quand tu fais retour en arrière du coup depuis la page des stocks)

Reply:
en fait j'ai l'impression que le redirect n'est pas appliqué quand le shouldBlock est à true, du coup ça revient au return true à la fin

Reply:
non c'est une erreur de ma part, je corrige, ça renvoie bien sur offres